MAUI/DataForm/localization.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: Data localization in .NET MAUI DataForm control | Syncfusion
+description: Learn about the data localization support in Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Localization in .NET MAUI DataForm (SfDataForm)
+
+Localization is the process of translating the application resources into different language for the specific cultures. The `SfDataForm` can be localized by adding `resource` file. 
```

Comment:
Localization is the process of translating the application resources into a different language for specific cultures. The `SfDataForm` can be localized by adding a `resource` file.